static/app/components/repositoryRow.tsx

```diff
@@ -165,10 +219,16 @@ const RepositoryTitleAndUrl = styled('div')`
   flex-direction: column;
 `;
 
+const EditAndDelete = styled('div')`
+  display: flex;
+  margin-left: ${space(1)};
+`;
+
 const RepositoryTitle = styled('div')`
   margin-bottom: ${space(1)};
   /* accommodate cancel button height */
   line-height: 26px;
 `;
 
-export default RepositoryRow;
+export default withOrganization(RepositoryRow);
+export {RepositoryRow};
```

Comment:
@MeredithAnya is there a case we want this? This potentially causes RepositoryRow to have props.organization undefined

Reply:
@NisanthanNanthakumar I had this for tests but realized I wasn't even using it imma update the tests